app/Http/Resources/OneChoiceQuestion.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use App\Http\Resources\Option as OptionShortResource;
+
+class OneChoiceQuestion extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'topic_id' => $this->topic_id,
+            'type' => $this->type,
```

Comment:
Remove this property from the resource, as it's already a resource for the one-choice questions.